### ISSUE TITLE
Verify / Fix out of source builds

### ISSUE THREAD

**Issue by paul-pearce** [zmap/zmap#407]
It's unclear if out of source builds are currently working. Test, fix as necessary.

**Comment by zakird** [comment]
Is this still an issue?

**Comment by dadrian** [comment]
I don't know if it builds out of source or not, but regardless we should do an out of source build on TravisCI to make sure it keeps working.

**Comment by ogasser** [comment]
Out of source builds still fail:
```
CMake Error: Error processing file: CMakeVersion.txt
make[2]: *** [src/CMakeFiles/git_versioning.dir/build.make:57: src/CMakeFiles/git_versioning] Error 1
make[1]: *** [CMakeFiles/Makefile2:336: src/CMakeFiles/git_versioning.dir/all] Error 2
make: *** [Makefile:150: all] Error 2
```

**Comment by paul-pearce** [comment]
Fixed by #425
